### PR TITLE
Blog App: Forms

### PR DESCRIPTION
**Pull request by jasonfelice**
# Rails Blog App: Milestone 7

## Forms
- [x] Created a method `current_user` in the ApplicationController
- [x] Created `CommentsController` & `LikesController`
- [x] Added forms in respective view

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class CommentsController < ApplicationController
+  def create
+    post = Post.find(params[:id])
+    @comment = current_user.comments.new(text: params[:comment])
+    @comment.post_id = post.id
+    @comment.save
+    redirect_to "/user/#{current_user.id}/post/#{post.id}"
+  end
```
Comment (VICTOR-CHUKWU) [inline app/controllers/comments_controller.rb]: - [x] when i submit a comment, it does not get added to the post, this is also same as when I like a post, kindly fix it
Screenshot from 2022-08-26 15-38-34

File: app/controllers/likes_controller.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class LikesController < ApplicationController
+  def create
+    post = Post.find(params[:post_id])
+    @like = post.like.new
+    @like.post_id = post.id
+    @like.author_id = current_user.id
+    @like.save
+    redirect_to "/user/#{current_user.id}/post/#{post.id}"
+  end
+end
```
Comment (VICTOR-CHUKWU) [inline app/controllers/likes_controller.rb]: - [x] Likes are not being added to post when clicked, kindly fix this too

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @jasonfelice  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by andersonlebon — APPROVED** [review]
Hi @jasonfelice 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is now professional :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**